composer.json

```diff
@@ -39,7 +39,7 @@
     "trntv/cheatsheet": "^0.1@dev",
     "trntv/yii2-command-bus": "^3.0",
     "intervention/image": "^2.1",
-    "vlucas/phpdotenv": "^2.0",
+    "vlucas/phpdotenv": "@stable",
```

Comment:
this is not recommended, since breaking changes may be introduced between major versions.